api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -108,15 +204,9 @@ def draft_registration(self, user, project_public, schema):
             branched_from=project_public
         )
 
-    @pytest.fixture()
-    def url_draft_registrations(self, project_public):
-        # Specifies version to test functionality when using DraftRegistrationLegacySerializer
-        return '/{}nodes/{}/draft_registrations/?{}'.format(
-            API_BASE, project_public._id, 'version=2.19')
-
-    def test_admin_can_view_draft_list(
-            self, app, user, draft_registration, project_public,
-            schema, url_draft_registrations):
+    def test_draft_admin_can_view_draft_list(
```

Comment:
`user` here is *also* a Project admin, as they are the `creator` of `project_public`.